core/src/main/java/org/stellar/anchor/sep31/Sep31Service.java

```diff
@@ -326,9 +326,13 @@ void validatePatchTransactionFields(Sep31Transaction txn, Sep31PatchTransactionR
     }
   }
 
-  void preValidateQuote() throws AnchorException {
+  void preValidateQuote(boolean isQuotesRequired) throws AnchorException {
```

Comment:
`assetInfo` is available by calling `Context.get().getAsset();` WDYT if we remove `boolean isAuotesRequired`? 

`
void preValidateQuote() throws AnchorException {
    AssetInfo assetInfo = Context.get().getAsset();
    boolean isQuotesRequired = assetInfo.getSep31().isQuotesRequired();
....
}

`

Reply:
Thanks for the suggestion, I've made this change